Makefile

```diff
@@ -101,4 +104,5 @@ check: checkcommits log-parser
 	log-parser \
 	openshift \
 	swarm \
+	vm-factory \
```

Comment:
ah, we tend to keep these in alphabetical order ... if you don't mind shuffling your addition to the end?